test/bigtest/interactors/check-in.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable */
```

Comment:
As I understand this is in order to avoid the one class per file problem. I would suggest disabling the rule in the eslint config. https://github.com/folio-org/ui-checkin/blob/master/test/bigtest/.eslintrc.js

Reply:
Splitted to separate classes